### PR TITLE
ci: add dco check

### PR DESCRIPTION
**Pull request by EthanPERRUZZA**

### CODE REVIEW

**Comment by emersion** [comment]
Could we maybe merge this new check with the existing `check_commits` job? It's pretty similar and it would avoid spinning up two containers.

**Review by emersion — APPROVED** [review]
LGTM :)

**Review by younesschrifi — APPROVED** [review]
LGTM ! 😁

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -285,6 +285,10 @@ jobs:
 
         commit_titles | TERM=xterm-color .github/scripts/check-commit-titles.sh
 
+    - name: Check DCO
+      uses: christophebedard/dco-check@0.5.0
+      env:
+        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```
Comment (Khoyo) [inline .github/workflows/build.yml]: We should exclude dependabot from the check (eg using `--exclude-emails '49699333+dependabot[bot]@users.noreply.github.com'`)

**Review by Khoyo — CHANGES_REQUESTED** [review]

File: .github/workflows/build.yml (hunk 1 above)
Comment (EthanPERRUZZA) [reply .github/workflows/build.yml]: done ;)

**Review by Khoyo — APPROVED** [review]

**Pull request merged**